lib/tasks/superbolt.rake

```diff
@@ -1,7 +1,13 @@
 # assumes an environment task that sets up your app environment
 desc "worker that reads the queue and sends messages through the router as configured"
 task :superbolt => :environment do
-  Superbolt::App.new(Superbolt.app_name, {}).run do |message, logger|
-    Superbolt::Router.new(message, logger).perform
+  begin
+    Superbolt::App.new(Superbolt.app_name, {}).run do |message, logger|
+      Superbolt::Router.new(message, logger).perform
+    end
+  # Rescue SignalException so that we don't spam error handlers like Airbrake
+  # with SIGTERM errors every time a task is killed (Heroku dyno restarts, etc.)
+  rescue SignalException => e
```

Comment:
The prob is that anytime someone kills superbolt.rake with a sigterm, it raises a Signal Exception (interestingly, it does not do so when someone kills with a sigkill).  What happens is that any user of this gem will then get those server errors.  It's currently spamming all the harvesters because whenever Heroku restarts dynos (which is frequently + when we do deploys), it sigterms everything running. Presumably when we move to AWS the same issue will be there, although hopefully we won't see machine restarts as often.